h/admin/services/user.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
I created it as an admin service specifically to signal that the implementation works but should probably be revisited before we move this out into a `h.accounts.services` and give this ability to users to rename their own account. For all the reasons we already touched on in the comments of this PR (it's at the moment impossible to show any progress to the user, etc.)